core/src/main/scripts/importer/metaImport.py

```diff
@@ -58,7 +58,7 @@ def interface():
                                        help='Skip tests requiring information '
                                             'from the cBioPortal installation')
     parser.add_argument('-jar', '--jar_path', type=str, required=False,
-                        help='Path to core JAR file')
+                        help='Path to scripts JAR file')
```

Comment:
Great, this fixes the issue I ran into, by documenting that this argument was affected by c594d70676fe136fba85ae8172307ef53a3353ac as well.